designer/client/src/Component.tsx

```diff
@@ -190,6 +190,30 @@ export const Html: FunctionComponent = () => {
   )
 }
 
+export const FileUploadField: FunctionComponent = () => {
+  return (
+    <ComponentField>
+      <div className="box">
+        <div className="fileUploadIconContainer" aria-hidden="true">
+          <svg
```

Comment:
Mind sticking this alongside **components/Icons/SearchIcon.tsx**?

Reply:
Done